DQM/TrackingMonitorSource/python/TrackCollections2monitor_cff.py

```diff
@@ -180,13 +180,84 @@
 doEffFromHitPatternVsBX             ['highPurityPt1'] = cms.bool(True)
 doStopSource                        ['highPurityPt1'] = cms.bool(True)
 
+###### all tracks (no pt cut) associated to the PV
+###### association is dz<1mm 
+from CommonTools.RecoAlgos.TrackWithVertexSelector_cfi import *
+
+trackAssociated2pvSelector = trackWithVertexSelector.clone()
+trackAssociated2pvSelector.trackWithVertexSelectorParams = cms.PSet(
+   # the track collection
+   src = cms.InputTag('generalTracks'),
+   # kinematic cuts  (pT in GeV)
+   etaMin = cms.double(0.0),
+   etaMax = cms.double(5.0),
+   ptMin = cms.double(0.0),
+   ptMax = cms.double(100000.0),
+   # impact parameter cut (in cm)
+   d0Max = cms.double(999.),
+   dzMax = cms.double(999.),
+   # quality cuts (valid hits, normalized chi2)
+   normalizedChi2         = cms.double(999999.),
+   numberOfValidHits      = cms.uint32(0),
+   numberOfLostHits       = cms.uint32(999), ## at most 999 lost hits
+   numberOfValidPixelHits = cms.uint32(0), ## at least <n> hits in the pixels
+   ptErrorCut             = cms.double(9999999.), ## [pTError/pT]*max(1,normChi2) <= ptErrorCut
+   quality = cms.string("highPurity"), # quality cut as defined in reco::TrackBase
+   # compatibility with a vertex ?
+   useVtx       = cms.bool(True),
+   vertexTag    = cms.InputTag('trackingDQMgoodOfflinePrimaryVertices'),
+   timesTag     = cms.InputTag(''),
+   timeResosTag = cms.InputTag(''),
+   nVertices    = cms.uint32(0), ## how many vertices to look at before dropping the track
```

Comment:
`nVertices == 0` disables the vertex compatibility checks, i.e. only [`TrackWithVertexSelector::testTrack()`](https://github.com/cms-sw/cmssw/blob/CMSSW_9_0_X/CommonTools/RecoAlgos/src/TrackWithVertexSelector.cc#L51) gets called.